README.md

```diff
@@ -42,32 +37,13 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
```

Comment:
[OPTIONAL] I recommend you change `Author1` to your full name. It'll make your documentation look more professional.